compiler-cli/src/new.rs

```diff
@@ -142,6 +151,7 @@ pub fn hello_world_test() {
             Self::GleamToml => Some(format!(
                 r#"name = "{project_name}"
 version = "1.0.0"
+{target}
```

Comment:
This outputs an extra empty line for Erlang. Let's keep the Erlang format as it was before.

Reply:
I've corrected the template in aa39651a7ab41df0a1acb7754c4ab00b06df393c and 19a4695e75f89e215c2fb3f332705eb8747dcbd4 and the template now looks as expected.